### PR TITLE
Hearing Worksheet : Persist Issues Descriptions to Store

### PR DESCRIPTION
**Pull request by nolawi**
Connects #2990 

screen shot 2017-08-31 at 7 37 32 am

### CODE REVIEW

File: client/app/hearings/HearingWorksheet.jsx [hunk 1]
```diff
@@ -69,42 +70,16 @@ export class HearingWorksheet extends React.PureComponent {
...
-        ]
-      },
-      {
-        program: 'Compensation',
-        issue: 'Service connection',
-        levels: 'All Others, 8100 - Migraine',
-        description: 'Frequent headaches',
-        actions: [
-          false, false, false, false, false, false
+          false, false, true, true, false, false
```
Comment (joofsh) [inline client/app/hearings/HearingWorksheet.jsx]: Is there a reason we got rid of the extra issues?

File: client/app/hearings/reducers/index.js [hunk 2]
```diff
@@ -62,6 +62,11 @@ export const hearingsReducers = function(state = mapDataToInitialState(), action
   case Constants.SET_TRANSCRIPT_REQUESTED:
     return newHearingState(state, action, { transcript_requested: { $set: action.payload.transcriptRequested } });
 
+  case Constants.SET_DESCRIPTIONS:
+    return update(state, {
+      worksheet: { streams: { description: { $set: action.payload.description } } }
```
Comment (joofsh) [inline client/app/hearings/reducers/index.js]: how would this work with multiple appeals & multiple issues? I think `streams` will need to be an array, and `issues` an array within each appeal

File: client/app/hearings/HearingWorksheet.jsx (hunk 1 above)
Comment (nolawi) [reply client/app/hearings/HearingWorksheet.jsx]: The whole thing is a placeholder, if its preferred I can put it back but its confusing till we have an `id` for the issue; trying to keep PR's smaller

**Comment by nolawi** [comment]
@joofsh I have udpated this based on the structure from `txt` file..

**Pull request merged**